src/main/webapp/js/nav.js

```diff
@@ -0,0 +1,91 @@
+const loadingScreen = document.querySelector('.loading-screen')
+const mainNavigation = document.querySelector('.main-navigation')
+
+// Function to add and remove the page transition screen
+function pageTransitionIn() {
+    // GSAP methods can be chained and return directly a promise
```

Comment:
remove these comments throughout this file

Reply:
done